Common/include/option_structure.inl

```diff
@@ -1772,6 +1772,106 @@ class COptionActDisk : public COptionBase {
   }
 };
 
+class COptionActDiskBem : public COptionBase {
+  string name;  // identifier for the option
+  unsigned short& inlet_size;
+  unsigned short& outlet_size;
+  string*& marker_inlet;
+  string*& marker_outlet;
+  su2double**& dir_x;
+  su2double**& dir_y;
+  su2double**& dir_z;
+
+ public:
+  COptionActDiskBem(const string name, unsigned short& nMarker_ActDiskInlet, unsigned short& nMarker_ActDiskOutlet,
+                    string*& Marker_ActDiskInlet, string*& Marker_ActDiskOutlet,
+                    su2double**& ActDiskBem_X, su2double**& ActDiskBem_Y, su2double**& ActDiskBem_Z)
```

Comment:
I think this does the same as COptionActDisk but just with renamed variables right? If so try to re-use COptionActDisk instead of introducing this new option.